### PR TITLE
fix spelling errors and pep8 violation

### PR DESCRIPTION
**Pull request by 5pence**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Steps:
 
 ### Bump the lib Version
 
-Check the last release number at https://pypi.org/manage/project/scanapi/releases/
```
Comment (camilamaia) [inline CONTRIBUTING.md:113]: Any special reason for removing the url? Or was it just a mistake?

File: scanapi/errors.py [hunk 2]
```diff
@@ -5,9 +5,9 @@ class MalformedSpecError(Exception):
 class HTTPMethodNotAllowedError(MalformedSpecError):
     """Raised when the HTTP method in the API spec is invalid"""
 
-    def __init__(self, method, allowed_methos, *args):
+    def __init__(self, method, allowed_methods, *args):
```
Comment (camilamaia) [inline scanapi/errors.py:8]: 👀 👏

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.6] - 2020-07-25
```
Comment (camilamaia) [inline CHANGELOG.md:10]:
```suggestion

```
We keep all the changes under `Unreleased` section and only when we want in fact to create a new release, we move them to the proper release title, like this https://github.com/scanapi/scanapi/pull/212/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10

So for now, we can leave everything under ##[Unreleased]

File: scanapi/reporter.py [hunk 4]
```diff
@@ -29,7 +29,8 @@ def write(self, results):
         logger.info("\nThe documentation was generated successfully.")
         logger.info(f"It is available at {self.output_path}")
 
-    def _build_context(self, results):
+    @staticmethod
+    def _build_context(results):
```
Comment (camilamaia) [inline scanapi/reporter.py:33]: Sorry, I did not get the idea of changing this method to a static one. Would you mind to give me some context?
And I believe we would need to change the related calls too.

File: CONTRIBUTING.md (hunk 1 above)
Comment (5pence) [reply CONTRIBUTING.md]: opps mistake :)

**Pull request merged**